Clock It/StatefulView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  PrimaryView.swift
+//  Clock It
+//
+//  Created by Eric Masiello on 10/31/24.
+//
+
+import SwiftUI
+
+struct StatefulView: View {
```

Comment:
Somewhere in this file we need to do the thing that disables the screen from turning off